fe/fe-core/src/main/java/org/apache/doris/analysis/StmtRewriter.java

```diff
@@ -1141,5 +1147,49 @@ private static Expr createJoinConjunct(Expr exprWithSubquery, InlineViewRef inli
         smap.put(subquery, subquerySubstitute);
         return exprWithSubquery.substitute(smap, analyzer, false);
     }
+    
+    public static void rewriteByPolicy(StatementBase statementBase, Analyzer analyzer) throws UserException {
+        Catalog currentCatalog = Catalog.getCurrentCatalog();
+        if (!(statementBase instanceof SelectStmt)) {
+            return;
+        }
+        SelectStmt selectStmt = (SelectStmt) statementBase;
+        for (int i = 0; i < selectStmt.fromClause_.size(); i++) {
+            TableRef tableRef = selectStmt.fromClause_.get(i);
+            if (tableRef instanceof InlineViewRef) {
+                InlineViewRef viewRef = (InlineViewRef) tableRef;
+                rewriteByPolicy(viewRef.getQueryStmt(), analyzer);
+            }
+            // has been rewrite
+            if (tableRef instanceof InlineViewRef) {
+                continue;
```

Comment:
Why not move this `continue` to to above `if (tableRef instanceof InlineViewRef) {`?

Reply:
I'm going to modify it